openmdao/docs/features/building_components/explicitcomp.rst

```diff
@@ -16,6 +16,13 @@ ExplicitComponent Methods
 
 The implementation of each method will be illustrated using a simple explicit component that computes the output *area* as a function of inputs *length* and *width*.
 
+::
```

Comment:
I didn't mean to hold up the pull request for this. Let's just merge and clean up formatting later when we do a pass through all the docs